### DISCUSSION TITLE
Can IDOM "lazy-load" parts of the webpage?

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#359] [Question] [answered]
How would you recommend doing lazy load components within IDOM? For example, loading the IDOM viewport then immediately loading another component on the page asynchronously.

**Comment by rmorshea** [comment, marked as answer]
If you need to update things imperatively [`hotswap`](https://idom-docs.herokuapp.com/docs/package-api.html#idom.widgets.utils.hotswap) is probably what you'd want.
